importer/tests/test_import.py

```diff
@@ -199,28 +203,46 @@ def test_import_static_recurse(self):
         )
         self.addCleanup(default_storage.delete, asset.asset)
 
-    def test_static_import_integration(self):
-        """
-        Do integration test to validate course import with static assets.
-        """
-        original_count = Course.objects.count()
-        tarball_file = self.get_course_single_tarball()
-        import_file(
-            tarball_file, self.repo.id, self.user.id)
-        self.assertEqual(
-            Course.objects.count(),
-            original_count + 1,
+    def test_parse_static(self):
+        """
+        Parse the static assets in the sample course
+        """
+        def get_counts():
+            """Returns counts of resources, videos, and assets."""
+            counts = namedtuple("counts", "resources videos assets")
+            kwargs = {"course__course_number": "toy"}
+            resources = LearningResource.objects.filter(**kwargs).count()
+            assets = StaticAsset.objects.filter(**kwargs).count()
+            kwargs["learning_resource_type__name"] = "video"
+            videos = LearningResource.objects.filter(**kwargs).count()
+            return counts(resources, videos, assets)
+
+        # There should be nothing.
+        counts = get_counts()
+        self.assertTrue(counts.resources == 0)
+        self.assertTrue(counts.videos == 0)
+        self.assertTrue(counts.assets == 0)
+        # Import the course.
+        import_course_from_file(
+            self.get_course_single_tarball(),
+            self.repo.id, self.user.id
         )
-        self.assertFalse(os.path.exists(tarball_file))
-        course = Course.objects.all().exclude(id=self.course.id)
-        self.assertEqual(course.count(), 1)
-        assets = StaticAsset.objects.filter(course=course)
-        for asset in assets:
-            self.addCleanup(default_storage.delete, asset.asset)
-        self.assertEqual(assets.count(), 2)
-        for asset in assets:
-            base_path = static_asset_basepath(asset, '')
-            self.assertIn(
-                asset.asset.name.replace(base_path, ''),
-                ['test.txt', 'subdir/subtext.txt']
```

Comment:
The cleanup is happening in the `LoreTestCase.tearDown()`. It seems like a good place to cover all static assets instead of calling `addCleanup()` in individual tests. Any objections?